mysql-test/suite/innodb/r/innodb_stats_fetch.result

```diff
@@ -150,3 +150,8 @@ max_data_length	0
 index_length	16384
 DROP TABLE test_ps_fetch;
 set @@use_stat_tables = @save_use_stat_tables;
+#
+# MDEV-28613 LeakSanitizer caused by I_S query using LIMIT ROWS EXAMINED
+#
+SELECT * FROM INFORMATION_SCHEMA.INNODB_SYS_INDEXES LIMIT ROWS EXAMINED 5;
+ERROR HY000: Unknown error
```

Comment:
Can we get an outcome that is similar to the following (tested on 10.5 c638051d808a7b29b10e231c9eae97d5909d17f6)?
```
SELECT * FROM seq_1_to_100 limit rows examined 5;
seq
1
2
Warnings:
Warning	1931	Query execution was interrupted. The query exceeded LIMIT ROWS EXAMINED 5. The query result may be incomplete
```
Please make the test case cover all tables whose handling was revised.